src/main/java/thePackmaster/cards/quietpack/Frontflip.java

```diff
@@ -0,0 +1,36 @@
+package thePackmaster.cards.quietpack;
+
+
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.actions.defect.ScrapeFollowUpAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.actions.quietpack.FrontflipAction;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+
+public class Frontflip extends AbstractPackmasterCard {
+    public final static String ID = makeID("Frontflip");
+
+    public Frontflip() {
+        super(ID, 2, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.SELF);
+        baseBlock = 13;
+        magicNumber = baseMagicNumber = 3;
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        blck();
+        this.addToBot(new DrawCardAction(this.magicNumber, new FrontflipAction()));
```

Comment:
```suggestion
        addToBot(new DrawCardAction(magicNumber, new AbstractGameAction() {
            @Override
            public void update() {
                Wiz.att(new WaitAction(3));
                DrawCardAction.drawnCards.stream().filter(c -> c.cost == -2).forEach(c -> Wiz.att(new DiscardSpecificCardAction(c)));

            }
        }));
```
No need for the extra action.
Still, why the long wait that happens after the discard?